.github/workflows/PRBuild.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         $CommitNumber = git rev-list --no-merges --count HEAD
 
         dotnet build dotnet\DotNetStandardClasses.sln --no-restore --configuration $Env:Configuration /p:CommitNumber=$CommitNumber
-        echo "::set-env name=CommitNumber::$CommitNumber"
+		echo "CommitNumber=$CommitNumber" >> $env:GITHUB_ENV
```

Comment:
Identation is VERY important at .yml files. Please, use spaces instead of tabs.

Reply:
fixed